pkg/requests/examples.go

```diff
@@ -86,25 +74,25 @@ func (ex *Examples) chargeCreated(card string, data []string) (map[string]interf
 // sends another request to specifically capture it to trigger the
 // captured event
 func (ex *Examples) ChargeCaptured() error {
-	charge, err := ex.chargeCreated("4242424242424242", []string{
+	charge, err := ex.chargeCreated(dummyToken, []string{
 		"amount=2000",
 		"currency=usd",
 		"capture=false",
 	})
+
 	if err != nil {
 		return err
 	}
 
 	req, params := ex.buildRequest(http.MethodPost, []string{})
 	reqURL := fmt.Sprintf("/v1/charges/%s/capture", charge["id"])
-
 	_, err = ex.performStripeRequest(req, reqURL, params)
 	return err
 }
 
 // ChargeFailed fails to create a charge
 func (ex *Examples) ChargeFailed() error {
-	_, err := ex.chargeCreated("4000000000000002", []string{
+	_, err := ex.chargeCreated(dummyToken, []string{
```

Comment:
`tok_visa` (the value of `dummyToken`) is equivalent to the `4242424242424242` test number that results in a successful charge, so you can't use for the `ChargeFailed` function.

The test token that matches the `4000000000000002` test number (that results in a failed charge) is `tok_chargeDeclined` (cf. https://stripe.com/docs/testing#cards-responses).

Reply:
🤦‍♂ fixing, sorry

Reply:
pushed fix to remote fork